### DISCUSSION TITLE
Cluster render does not show all markers when zoom limit is reached

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#831] [Q&A] [answered]
The cluster renderer (ClusterMarkerOverlayActivity) sometimes does not show all markers when zoom limit is reached: https://youtu.be/JR2ff-mEJtQ

**Comment by devemux86** [comment, marked as answer]
It is in the experiments section.

(could be a precision case)

**Reply by demiantres** [reply]
Ok, I just wanted to report it.
